yarn.lock

```diff
@@ -22348,6 +22348,14 @@ libphonenumber-js@^1.7.22:
     minimist "^1.2.0"
     xml2js "^0.4.17"
 
+libphonenumber-js@^1.7.50:
```

Comment:
Since there are no removals in yarn.lock, some other packages must still be relying on google-phone-lib. Would be good to double check that